### ISSUE TITLE
detect return to center on axis

### ISSUE THREAD

**Issue by arendeutsch** [ArunMichaelDsouza/joypad.js#11]
Hello,
Is it possible to detect return to center on analog stick ?

**Comment by ArunMichaelDsouza** [comment]
Hi @arendeutsch,

You can check for the [axisMovementValue](https://github.com/ArunMichaelDsouza/joypad.js/blob/7ceb11052abbae153ace9eccd09a8059902db793/src/events.js#L100) parameter returned from the ``axis_move`` event. This will give you the value of the axis movement. The center would be close to ``0``.

```javascript
joypad.on('axis_move', e => {
  console.log(e.detail.axisMovementValue);
});
```

Please note some older/used controllers might not go all the way to 0.

**Comment by arendeutsch** [comment]
Yes, im not sure it happens,
When you release the analog stick you dont get an event fire up when it closed to zero.
If i pull one of the stick all the way which give 1 or -1 and release then i might give 0.8 or 0.9 but not close to zero

**Comment by ArunMichaelDsouza** [comment]
Yes, that is correct. No event is fired when the stick value comes close to zero. Will have to look into this.
